tests/conftest.py

```diff
@@ -0,0 +1,160 @@
+    metadata.tables["ledger"].columns["recipient"].type = Text()
+
+    async with engine.begin() as conn:
+        await conn.run_sync(metadata.create_all)
+    yield
+    async with engine.begin() as conn:
+        await conn.run_sync(metadata.drop_all)
+
+
+@pytest_asyncio.fixture(scope="function")
+async def seed_users():
+    """Seed the test database with users."""
+    async with Session.begin() as session:
+        test_users = [
+            {
+                "id": "1",
+                "name": "bestower",
+                "discord_snowflake": "1",
+                "is_active": True,
+                "point_giver": True,
+            },
+            {
+                "id": "2",
+                "name": "recipient",
+                "discord_snowflake": "2",
+                "is_active": True,
+                "point_giver": True,
+            },
+            {
+                "id": "3",
+                "name": "recipient2",
+                "discord_snowflake": "3",
+                "is_active": True,
+                "point_giver": True,
+            },
+            {
+                "id": "4",
+                "name": "abba-zaba",
+                "discord_snowflake": None,
+                "is_active": True,
+                "point_giver": False,
+            },
+            {
+                "id": "5",
+                "name": "mcdonald's",
+                "discord_snowflake": None,
+                "is_active": True,
+                "point_giver": False,
+            },
+            {
+                "id": "6",
+                "name": "name with spaces",
+                "discord_snowflake": None,
+                "is_active": True,
+                "point_giver": False,
+            },
+            {
+                "id": "7",
+                "name": "bestower_inactive",
+                "discord_snowflake": "7",
+                "is_active": False,
+                "point_giver": True,
+            },
+            {
+                "id": "8",
+                "name": "bestower_non_point_giver",
+                "discord_snowflake": "8",
+                "is_active": True,
+                "point_giver": False,
+            },
+            {
+                "id": "9",
+                "name": "recipient_inactive",
+                "discord_snowflake": "9",
+                "is_active": False,
+                "point_giver": True,
+            },
+            {
+                "id": "10",
+                "name": "recipient_non_point_giver",
+                "discord_snowflake": "10",
+                "is_active": True,
+                "point_giver": False,
+            },
+        ]
+        await session.execute(pzsd_user.insert().values(test_users))
+
+
+@pytest.fixture
+def mock_bot():
+    mock_bot = MagicMock()
+
+    mock_channel = AsyncMock()
+    mock_channel.send = AsyncMock()
+
+    mock_bot.get_channel.return_value = mock_channel
+
+    return mock_bot
+
+
+@pytest.fixture
+def mock_db_session():
```

Comment:
not used

Reply:
oh wait, it's using patch, but then how is the sqlite db being used then

Reply:
I use this fixture in the last test (`test_excessive_point_violation`)

Reply:
the rest of the tests handle all db situations, this test really just needed to make sure that the point amount outside the allowed range doesn't result in calling the `bestow_points` method